### PR TITLE
Verifier sample using React w/ CRA

### PR DESCRIPTION
**Pull request by tmarkovski**
This is the extent of my react skills

### CODE REVIEW

File: browser/verifier-oidc4vp-cra/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "verifier-oidc4vp-cra",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
```
Comment (janpieterz) [inline browser/verifier-oidc4vp-cra/package.json:6]: For clarity I've moved non-essential dependencies in other samples to `devDependencies`

File: browser/verifier-oidc4vp-cra/src/AppContent.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  public authService: AuthService;
+
+  constructor(props: any) {
+    super(props);
+
+    this.authService = new AuthService();
+    this.state = { token: "" };
+  }
+
+  public componentDidMount() {
```
Comment (janpieterz) [inline browser/verifier-oidc4vp-cra/src/AppContent.tsx:16]: This is quite old-style React, could we refactor to hooks / functional component?

**Review by janpieterz — APPROVED** [review]
Oidc client, could we take that (and it's ts version) as an npm dependency instead?

And maybe the callback routed through React as well?

File: browser/verifier-oidc4vp-cra/src/AppContent.tsx (hunk 2 above)
Comment (tmarkovski) [reply browser/verifier-oidc4vp-cra/src/AppContent.tsx:16]: Contributions are welcome :)

**Comment by janpieterz** [comment]
Ready for your merge (maybe do a double check, I did and it works)

**Pull request merged**